Net SDK Unit Tests/Unit Tests/Matchers/SplitParserUnitTests.cs

```diff
@@ -21,7 +21,7 @@ public void ParseSuccessfullyWhenNonSpecifiedAlgorithm()
             Split split = new Split();
             split.name = "test1";
             split.seed = 2323;
-            split.status = StatusEnum.ACTIVE;
+            split.status = "ACTIVE";
```

Comment:
Yes, that was causing exceptions when trying to set an invalid value (not defined in enum). This way, when deserializing splits, it will map the value as string, and then the parser will be responsible to map that string to enum. 
Also, it prevents from future exceptions when setting unknown values for enums.